src/esn.calendar.libs/app/services/dav-request.js

```diff
@@ -13,11 +13,15 @@ function calDavRequest($http, $q, tokenAPI, calCalDAVURLService, CAL_GRACE_DELAY
       delete params.graceperiod;
     }
 
-    return $q.all([tokenAPI.getNewToken(), _getDavServerUrl()])
-      .then(([{ data }, serverBaseUrl]) => {
+    return $q.all([tokenAPI.getNewToken(), _getDavServerUrl(), tokenAPI.getWebToken()])
```

Comment:
It bugs me to see that even though we're only using either ESNToken or JWT but we're sending requests to get both here... Why can't we just remove ESNToken? cc @renaudboyer @rezk2ll

Reply:
it's to ease the transition from ESNTOKEN to JWT. if JWT is not available we use ESNTOKEN

Reply:
Why would JWT not be available though?

Reply:
the public key can be missing from the server. the technical user making requests from the ESN backend etc...

Reply:
Okay